NuRadioMC/examples/06_webinar/W02RunSimulation.py

```diff
@@ -82,9 +82,12 @@ def _detector_simulation(self):
         efieldToVoltageConverter.run(self._evt, self._station, self._det)
         """
         Our simulation uses the default sampling rate of 5 GHz, or 5 GS/s, or
-        equivalently, a time step of 0.2 ns. Such a high resolution is not common
+        equivalently, a time step of 0.2 ns. Such a high resolution, while needed
+        during simulations to capture all the details of the radio wave, is not common
         at all in radio experiments, where the sampling rates tend to lie around
-        the gigahertz. So, if we want our simulation as realistic as possible,
+        the gigahertz. If we want our simulation to be a theoretical study, we
+        can use 5 GHz as the electric field sampling rate. However, if we want
+        to simulate what an actual radio detector would see,
```

Comment:
not correct. None of the detectors digitizes the trace and then runs a amplitude trigger.

Reply:
Well, yes. But a 1 or 2 GS/s trace is still going to be closer to what the ADC sees than a 5 GS/s trace, right?